### ISSUE TITLE
UI disappears abruptly

### ISSUE THREAD

**Issue by nikpydev** [FaisalUmair/udemy-downloader-gui#375]
The app UI disappears abruptly. Uninstalling and Re-Installing didn't work either.
![udeler_error](https://user-images.githubusercontent.com/44652898/75097578-4954aa80-55d2-11ea-97b4-cf45855f5068.png)

**Comment by kawaiiyoshiyuki** [comment]
I am experiencing the same, especially when clicked to download more than one course. It seems it gots stuck on attachment on one of the courses and then immediately fails completely, I had hard time to get rid of the frozen GUI, force quit didn't help, had to restart computer.

**Comment by askaleks** [comment]
I have the same issue, do you guys know how to fix this ?

**Comment by nikpydev** [comment]
It was my mistake actually. I had set the download location to an external disk but that very disk got disconnected due to loose connection and I didn't notice it at first. When I fixed that loose connection issue, things were back to normal almost instantly. Turns out the uninstalling and reinstalling wasn't even required. "CHECK YOUR DOWNLOAD LOCATION AND MAKE SURE IT IS ACCESSIBLE BY UDELER" and your problem will be resolved.
